NuGet.sln

```diff
@@ -208,8 +208,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestableCredentialProvider"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "API.Test", "test\TestExtensions\API.Test\API.Test.csproj", "{90AEABA2-0FB5-4D6C-A505-FA7F9ED38076}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestableVSCredentialProvider", "test\TestExtensions\TestableVSCredentialProvider\TestableVSCredentialProvider.csproj", "{7D0DDCB8-E362-4283-8AC8-3FF1E9B0263E}"
```

Comment:
We should remove the entry for this project in the below solution filter file. I found a reference to this file in https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20TestableVSCredentialProvider&type=code.

https://github.com/NuGet/NuGet.Client/blob/764d370df4d3c01c24340f57e6389058bce24ddf/NuGet-UnitTests.slnf#L72

Reply:
Good catch!